spec/intl.html

```diff
@@ -40,23 +40,77 @@ <h1><a href="https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping">Cas
 
     <ins class="block">
       <p>
-        <emu-xref href="#sec-time-zone-names"></emu-xref> defines a set of abstract operations concerning the names of supported time zones.
+        <emu-xref href="#sec-time-zone-support"></emu-xref> defines a set of abstract operations related to the identifiers of supported time zones.
         This section introduces additional requirements on these operations for implementations.
       </p>
+
+      <emu-clause id="sup-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          The ECMAScript Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons, but time zone identifiers returned by ECMAScript built-in objects always use the casing found in the IANA Time Zone Database.
+          A primary time zone identifier is a Zone name, and a non-primary time zone identifier is a Link name, respectively, in the IANA Time Zone Database except as specifically overridden by AvailableTimeZoneIdentifiers.
+          In an implementation that implements the ECMAScript Internationalization API Specification, available time zone identifiers are the set of Zone and Link identifiers that are supported by an ECMAScript implementation.
+        </p>
+
+        <p>
+          In the IANA Time Zone Database, the UTC time zone is represented by the Zone *"Etc/UTC"*, and is distinct from the Zone *"Etc/GMT"*.
+          For historical reasons, ECMAScript uses *"UTC"* as the primary identifier for the former Zone and does not recognize the latter Zone as distinct, instead requiring treatment of *"Etc/UTC"*, *"Etc/GMT"*, and *"GMT"* (if available) as non-primary identifiers that resolve to *"UTC"*.
+          This is the only deviation from the IANA Time Zone Database that is required of an ECMAScript implementation.
+        </p>
+
+        <p>
+          Time zone aware ECMAScript implementations support *"UTC"* and all other Zone and Link names (and <b>only</b> such names) from the IANA Time Zone Database, the mapping between Link names and corresponding Zone names, and the UTC offsets and transitions associated with Zone names that represent local political rules for that time zone.
+          An ECMAScript implementation that includes the ECMA-402 Internationalization API <b>must</b> be time zone aware.
+          However, it is recommended that all ECMAScript implementations be time zone aware, including those that do not implement the ECMA-402 Internationalization API.
+          Furthermore, even non-time zone aware implementations that do not support the entire IANA Time Zone Database are still recommended to use IANA Time Zone Database names as identifiers to represent time zones.
```

Comment:
These two sentences can be shortened a bit, or maybe cut altogether? This information about non-time-zone-aware implementations is already present in the ECMA-262 part, but also it's not relevant here in the ECMA-402 part because we've already said that ECMA-402 requires time zone awareness.

Reply:
I shortened to this: "Some implementations that do not implement the ECMA-402 Internationalization API may also be time zone aware." OK?